response_operations_ui/templates/conversation-view/cv-message-block.html

```diff
@@ -14,6 +14,13 @@
             Sent:
             <span name="sm-sent-date" id="sm-sent-date-{{ loop.index }}" style="display:inline;">{{ message.get('sent_date') }}</span>
         </span>
+        {% if loop.last %}
+          {% if show_mark_unread == True %}
+            <a class="secure-message-mark-unread" id="sm-mark-as-unread"
+               href={{ url_for('messages_bp.mark_message_unread',
```

Comment:
Do quotes need to be added to either side of the curly brackets, e.g.: `href="{{ url_for...`